src/js/tabs/firmware_flasher.js

```diff
@@ -119,132 +138,195 @@ TABS.firmware_flasher.initialize = function (callback) {
-                releaseData.forEach(function(release){
-                    release.assets.forEach(function(asset){
-                        var targetFromFilenameExpression = /betaflight_([\d.]+)?_?(\w+)(\-.*)?\.(.*)/;
-                        var match = targetFromFilenameExpression.exec(asset.name);
-
-                        if ((!showDevReleases && release.prerelease) || !match) {
-                            return;
-                        }
-                        var target = match[2];
-                        if($.inArray(target, unsortedTargets) == -1) {
-                            unsortedTargets.push(target);
-                        }
-                    });
-                    sortedTargets = unsortedTargets.sort();
+        function processBoardOptions(releaseData, showDevReleases) {
+            var releases = {};
+            var sortedTargets = [];
+            var unsortedTargets = [];
+            releaseData.forEach(function(release){
+                release.assets.forEach(function(asset){
+                    var targetFromFilenameExpression = /betaflight_([\d.]+)?_?(\w+)(\-.*)?\.(.*)/;
+                    var match = targetFromFilenameExpression.exec(asset.name);
+                    if ((!showDevReleases && release.prerelease) || !match) {
+                        return;
+                    }
+                    var target = match[2];
+                    if($.inArray(target, unsortedTargets) == -1) {
+                        unsortedTargets.push(target);
+                    }
                 });
-                sortedTargets.forEach(function(release) {
-                    releases[release] = [];
+                sortedTargets = unsortedTargets.sort();
+            });
+            sortedTargets.forEach(function(release) {
+                releases[release] = [];
+            });
+            releaseData.forEach(function(release){
+                var versionFromTagExpression = /v?(.*)/;
+                var matchVersionFromTag = versionFromTagExpression.exec(release.tag_name);
+                var version = matchVersionFromTag[1];
+                release.assets.forEach(function(asset){
+                    var targetFromFilenameExpression = /betaflight_([\d.]+)?_?(\w+)(\-.*)?\.(.*)/;
+                    var match = targetFromFilenameExpression.exec(asset.name);
+                    if ((!showDevReleases && release.prerelease) || !match) {
+                        return;
+                    }
+                    var target = match[2];
+                    var format = match[4];
+                    if (format != 'hex') {
+                        return;
+                    }
+                    var date = new Date(release.published_at);
+                    var formattedDate = ("0" + date.getDate()).slice(-2) + "-" + ("0"+(date.getMonth()+1)).slice(-2) + "-" + date.getFullYear() + " " + ("0" + date.getHours()).slice(-2) + ":" + ("0" + date.getMinutes()).slice(-2);
```

Comment:
Again, this is an old code, but if you can add spaces around `+` operator the code left will be better than the original.